inherited_resources.gemspec

```diff
@@ -1,5 +1,6 @@
-# -*- encoding: utf-8 -*-
-$:.push File.expand_path("../lib", __FILE__)
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
```

Comment:
The magic comment shouldn't be necessary these days... Also, [master's bundler](https://github.com/bundler/bundler/blob/fbb9c2dada3d98ce7932f9767ea3b2bdbba04eef/lib/bundler/templates/newgem/newgem.gemspec.tt#L6) uses `__dir__` here, which is a little simpler :)

Reply:
That's what I figured. Great, I will update it to follow that! Thanks!